packages/cli/src/installers/node/index.ts

```diff
@@ -144,7 +118,80 @@ export async function installNode(pkg: { [key: string]: any }) {
     )
   }
 
-  console.log(displayOutroMessage(pushApiKey, name))
+  const { method } = await inquirer.prompt([
+    {
+      type: "list",
+      name: "method",
+      message:
+        "Which method of configuring AppSignal in your project do you prefer?",
+      choices: [
+        "Using an appsignal.js configuration file.",
+        "Using system environment variables."
+      ],
+      default: "Using an appsignal.js configuration file."
+    }
+  ])
+
+  if (method == "Using an appsignal.js configuration file.") {
+    console.log()
+    console.log(`Writing ${configurationFilename} configuration file.`)
+
+    fs.writeFileSync(
+      path.join(dir, configurationFilename),
+      `const { Appsignal } = require("@appsignal/nodejs");
+
+const appsignal = new Appsignal({
+  active: true,
+  name: "${name}",
+  pushApiKey: "${pushApiKey}",
+});
+
+module.exports = { appsignal };`
+    )
+  }
+
+  console.log()
+
+  console.log(
+    `🎉 ${chalk.greenBright(
+      "Great news!"
+    )} You've just installed AppSignal to your project!`
+  )
+
+  console.log()
+
+  if (method == "Using an appsignal.js configuration file.") {
+    console.log(
+      `Now, you can run your application like you normally would, but use the --require flag to load AppSignal's instrumentation before any other library:`
+    )
+    console.log()
+    console.log(`    node --require './${configurationFilename}' index.js`)
+  } else {
+    console.log(`You've chosen to use environment variables to configure AppSignal:
+
+    export APPSIGNAL_PUSH_API_KEY="${pushApiKey}"
+
+If you're using a cloud provider such as Heroku etc., seperate instructions on how to add these environment variables are available in our documentation:
+
+ 🔗 https://docs.appsignal.com/nodejs/configuration
+
+Then, you'll need to initalize AppSignal in your app. Please ensure that this is done in the entrypoint of your application, before all other dependencies are imported!
+
+    const { Appsignal } = require(\"@appsignal/nodejs\");
+    
+    const appsignal = new Appsignal({
+      active: true,
+      name: \"${name}\"
```

Comment:
I don't think we should continue to suggest writing the initialisation code manually, in the application.

OpenTelemetry docs suggest always using `--require`, to make sure it's always imported first. And if we always tell users to do this, then we entirely eliminate a whole class of bugs that our users can run into, where they import a library in their code before its instrumentation code has been loaded.

I think it makes sense to *always* write the `appsignal.js` file ourselves and tell the user to `--require` it in their scripts. The only difference between the environment variables path and the config file path would be whether `pushApiKey` is present in the file.

Reply:
Done in https://github.com/appsignal/appsignal-javascript/pull/579/commits/e19a91a79c474740f5bcf24dc0dd9f70d0dd4098.